completions/dpkg

```diff
@@ -13,20 +13,20 @@ if _comp_have_command grep-status; then
 else
     _comp_xfunc_dpkg_installed_packages()
     {
-        command grep -A 1 "Package: ${1-}" /var/lib/dpkg/status 2>/dev/null |
-            command grep -B 1 -Ee "ok installed|half-installed|unpacked| \
-            half-configured" \
-                -Ee "^Essential: yes" |
-            awk "/Package: ${1-}/ { print \$2 }" 2>/dev/null
+        command awk -F '\n' -v RS='' "
+            \$1 ~ /Package: $1/ &&
+            \$2 ~ /ok installed|half-installed|unpacked|half-configured|^Essential: yes/ {
+                print(substr(\$1, 10));
+            }" /var/lib/dpkg/status 2>/dev/null
     }
 
     _comp_xfunc_dpkg_purgeable_packages()
     {
-        command grep -A 1 "Package: ${1-}" /var/lib/dpkg/status 2>/dev/null |
-            command grep -B 1 -Ee "ok installed|half-installed|unpacked| \
-            half-configured|config-files" \
-                -Ee "^Essential: yes" |
-            awk "/Package: ${1-}/ { print \$2 }" 2>/dev/null
+        command awk -F'\n' -vRS='' "
```

Comment:
Stylistic nit, we have `-F` and `-v` here without a following space, whereas in the above we have them with the space. Let's be consistent with one of the approaches.

Reply:
It should have the spaces here, as well. I must have missed this line.

```suggestion
        command awk -F '\n' -v RS='' "
```